package.json

```diff
@@ -3,45 +3,40 @@
   "version": "1.2.0",
   "description": "Markdown formatting buttons for text inputs.",
   "repository": "github/markdown-toolbar-element",
-  "main": "dist/index.umd.js",
-  "module": "dist/index.esm.js",
```

Comment:
Does Rollup do the right thing with a main + type key combination or do we still need the module key?